app/data/generators/degree.js

```diff
@@ -1,67 +1,86 @@
 const faker = require('faker')
 faker.locale = 'en_GB'
 
+const { DateTime } = require('luxon')
 const weighted = require('weighted')
-const degreeData = require('../degree')
+const DegreeHelper = require('../helpers/degrees')
 
-module.exports = (isInternationCandidate) => {
-  const item = () => {
-    const subject = faker.helpers.randomize(degreeData().subjects)
-    const predicted = faker.datatype.boolean()
-    const startDate = '2017'
-    const endDate = '2020'
+module.exports = (isInternationCandidate, dateOfBirth) => {
```

Comment:
Change to an object { isInternationalCandidate, dateOfBirth}.

You also have a typo.

Reply:
Fixed